### ISSUE TITLE
[MU4 Issue] Export not working on Manjaro Linux

### ISSUE THREAD

**Issue by esc430** [musescore/MuseScore#15789]
**Describe the bug**
I am running ` musescore-4.0-2 ` on Manjaro Linux. The updated is really nicely done but I am having one issue that ruins it's functionality. When I try to export anything (part/whole score/pdf/mscz) it does not work. The prompt box to save opens and I name the file but when I expect to get the exported file in the chosen directory there is nothing there. Are there some common trouble shooting tips any one has? Any advice would be appreciated

**To Reproduce**
1. Go to Export
2. Click on Export and answer the prompt
3. Go to your directory
4. There is no file there! 

**Expected behavior**
Exported files to show up

**Screenshots**
N/A

**Platform information**
 - OS: Manjaro Linux

**Additional context**
None

**Comment by MarcSabatella** [comment]
I'm not sure what "musescore-4.0-2" refers to, but the current version is 4.0.0, and the specific supported build for Linux is MuseScore-4.0.0-x86_64.AppImage.  Is that what you are using?  It is working fine for me for export.  Be sure the file name you specify is consistent with the file type (eg, the extension needs to be pdf for a PDF file, etc).

**Comment by Futurama39** [comment]
Here 4.0-2 is most likely the AUR build.

**Comment by MarcSabatella** [comment]
Sorry, I don’t know those terms, bit they sound like some sort of third party hoods.  Again, the supported build is the 4.0.0 AppImage you download directly from musescore.org, so if you’re experiencing issues with a third party build, best to switch to the official one.

**Comment by Futurama39** [comment]
To get back to the original point, I've tried replicating the issue on EndeavourOS (still arch-based so somewhat similar to the user). I've been unable to reproduce either on the AppImage nor on the AUR version. @esc430  could you verify that this pdf export...
1) doesn't work on very simple files as well (it could be some more complex issue with the score you're trying to export)
2) doesn't work on the AppImage version of the software
If you don't know how to launch the appimage version, you may need to make it executable with `chmod +x <file>` and then run it through the console with `./<file>`

**Comment by esc430** [comment]
@Futurama39 @MarcSabatella 

Sorry for the late response. I have been in the middle of first week of classes. I figured out the solution to my problem. When naming the file I need to add the file extension manually or it will not export. When exported a score as a pdf I needed to name it "test.pdf" for it to export. Nothing would happen if I named it just "test". I am not sure if this is just a me problem but I am super happy that it works now and thanks so much the suggestions!~

SOLUTION: Make sure to name file with file extension when exporting from Musescore. (Ex. test.pdf NOT test)
